### ISSUE TITLE
ML setup Sprint 1

### ISSUE THREAD

**Issue by matweds** [emilgq/MVK-GL#8]
### Sprint one
**Implement models**

- [x] XGBoost. 
- [x] RandomForest.  
- [x] Linear regression. 

**Setup hyperparameters**
> Tweakable by user later
- [x] Learning rate
- [x] Max Depth

* Try models on placeholder data
